### PR TITLE
doc: misc fixes for experimental features

### PR DESCRIPTION
**Pull request by joscollin**
Signed-off-by: Jos Collin <jcollin@redhat.com>

### CODE REVIEW

**Comment by joscollin** [comment]
jenkins render docs

File: doc/cephfs/experimental-features.rst [hunk 1]
```diff
@@ -44,12 +44,10 @@ some cases. Specifically, if you share a pool for multiple FSes and delete
 a snapshot in one FS, expect to lose snapshotted file data in any other FS using
 snapshots. See the :doc:`/dev/cephfs-snapshots` page for more information.
 
-Snapshots are known not to work with multi-MDS filesystems.
-
 For somewhat obscure implementation reasons, the kernel client only supports up
 to 400 snapshots (http://tracker.ceph.com/issues/21420).
 
-Snapshotting was blocked off with the "allow_new_snaps" flag prior to Firefly.
+Snapshotting was blocked off with the ``allow_new_snaps`` flag prior to Firefly.
```
Comment (ukernel) [inline doc/cephfs/experimental-features.rst]: snapshot is enabled by default. I think this line should be
Snapshotting was blocked off with the ``allow_new_snaps`` flag prior to Mimic.

Comment (joscollin) [reply doc/cephfs/experimental-features.rst]: @ukernel Done.

File: doc/cephfs/experimental-features.rst [hunk 2]
```diff
@@ -70,7 +68,7 @@ Note that snapshots and multiple filesystems are *not* tested in combination
 and may not work together; see above.
 
 Multiple filesystems were available starting in the Jewel release candidates
-but were protected behind the "enable_multiple" flag before the final release.
+but were protected behind the ``enable_multiple`` flag before the final release.
```
Comment (batrick) [inline doc/cephfs/experimental-features.rst]: `but must be turned on via the ... until declared stable.`

Comment (joscollin) [reply doc/cephfs/experimental-features.rst]: @batrick Done. Please check again. Thanks.

**Pull request merged**